pyccel/ast/core.py

```diff
@@ -601,7 +527,8 @@ class Assign(Basic):
 
     Examples
     --------
-    >>> from sympy import symbols, MatrixSymbol, Matrix
+    >>> from sympy import MatrixSymbol, Matrix
```

Comment:
Do we still use `sympy.MatrixSymbol` in Pyccel?

Reply:
No, I'm gonna locate and update all docstrings.

Reply:
done 71c33ad , 02c4dc9

Reply:
The matrix classes aren't used, but I think they have `_print_X` methods in all 4 printers

Reply:
Thanks!